PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -318,22 +318,161 @@ def miniAOD_customizeCommon(process):
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByTightIsolationMVArun2v1DBoldDMwLT, key = cms.InputTag("hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTraw","category"))
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByTightIsolationMVArun2v1DBoldDMwLT, toMultiplex = "hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTraw")
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByTightIsolationMVArun2v1DBoldDMwLT,
+                                   mapping = cms.VPSet(
+                                        cms.PSet(category = cms.uint32(0), cut = cms.string("RecoTauTag_tauIdMVAIsoDBoldDMwLT2017v1_WPEff60"), variable = cms.string("pt")),)
+                                    )
+    ## modify VTight WP
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByVTightIsolationMVArun2v1DBoldDMwLT, mvaOutput_normalization = "RecoTauTag_tauIdMVAIsoDBoldDMwLT2017v1_mvaOutput_normalization")
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByVTightIsolationMVArun2v1DBoldDMwLT, key = cms.InputTag("hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTraw","category"))
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByVTightIsolationMVArun2v1DBoldDMwLT, toMultiplex = "hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTraw")
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByVTightIsolationMVArun2v1DBoldDMwLT,
+                                    mapping = cms.VPSet(
+                                        cms.PSet(category = cms.uint32(0), cut = cms.string("RecoTauTag_tauIdMVAIsoDBoldDMwLT2017v1_WPEff50"), variable = cms.string("pt")),)
+                                    )
+    ## modify VVTight WP
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByVVTightIsolationMVArun2v1DBoldDMwLT, mvaOutput_normalization = "RecoTauTag_tauIdMVAIsoDBoldDMwLT2017v1_mvaOutput_normalization")
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByVVTightIsolationMVArun2v1DBoldDMwLT, key = cms.InputTag("hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTraw","category"))
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByVVTightIsolationMVArun2v1DBoldDMwLT, toMultiplex = "hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTraw")
+    run2_miniAOD_94XFall17.toModify(process.hpsPFTauDiscriminationByVVTightIsolationMVArun2v1DBoldDMwLT,
+                                    mapping = cms.VPSet(
+                                        cms.PSet(category = cms.uint32(0), cut = cms.string("RecoTauTag_tauIdMVAIsoDBoldDMwLT2017v1_WPEff40"), variable = cms.string("pt")),)
+                                    )
+    ## add VVLoose WP to PAT tau
+    _extTauIDSources = process.patTaus.tauIDSources.clone()
+    _extTauIDSources.byVVLooseIsolationMVArun2v1DBoldDMwLT = cms.InputTag("hpsPFTauDiscriminationByVVLooseIsolationMVArun2v1DBoldDMwLT")
+    run2_miniAOD_94XFall17.toModify(process.patTaus,
```

Comment:
similarly, modifications should be done where the module is defined, 
in PhysicsTools/PatAlgos/python/producersLayer1/tauProducer_cfi.py